snap/validate.go

```diff
@@ -47,6 +57,15 @@ func Validate(info *Info) error {
 		return err
 	}
 
+	epoch := info.Epoch
+	if epoch == "" {
+		return fmt.Errorf("snap epoch cannot be empty")
```

Comment:
This validation function is only called from `snap.ReadInfoFromSnapFile()`, so empty epochs from the store won't cause issues due to this function anyway. That said, I didn't touch the store side of things here since the epoch stuff isn't implemented there just yet. When it is, the epoch should be validated just as I assume the name is validated.